src/Nodes/Shapes/SVGRect.php

```diff
@@ -110,6 +112,47 @@ public function setHeight($height)
     }
 
 
+    /**
+     * @return string The rx coordinate of the upper left corner.
```

Comment:
rx/ry are for all corners, not just the upper left corner, right? So this doc comment is slightly misguiding (same for the other comments).